lib/index.js

```diff
@@ -227,7 +224,8 @@ api.set = async ({
   if(type === 'nonce') {
     // include old value in the query to ensure that no concurrent
     // changes occurred via another process
-    const oldValue = tokens.length > 0 ? tokens.slice() : {$in: [null, []]};
+    const oldValue = allTokens.length > 0 ? allTokens
+      .slice() : {$in: [null, []]};
```

Comment:
Split on conceptual parts of the statement if possible, it make it easier to read. That means, if your statement looks like this:

`<set this var equal to> <conditional> <this value if true> <this value if false>`

You'd break somewhere between the angled brackets, not within them -- unless there's no other choice.

```suggestion
    const oldValue = allTokens.length > 0 ?
      allTokens.slice() : {$in: [null, []]};
```